person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, name, age
```

Comment:
- [ ] Kindly make sure your variables are well written here because as you may notice after running this file you are getting these errors:
![image](https://user-images.githubusercontent.com/97100091/224777739-dedc8de1-5e68-4019-8089-6b00201aaf20.png)
- [ ] Also make sure the user can add the `parent_permission` as well, it's not necessarily going to be `true`
```rb
def initialize(age, name = 'Unknown', parent_permission= true) # Instead of
parent_permission: true
```
```suggestion
  attr_accessor :id, :name, :age
```